### ISSUE TITLE
[Bug]: Vehicle: Drive on water doesn't work

### ISSUE THREAD

**Issue by KernelFrog** [Yimura/YimMenu#3189]
### Describe the bug

The Vehicle: Drive on water option doesn't work.
After selecting this option, a vehicle (e.g. Vagrant) doesn't drive on the water.  It drives into the water and then stops.

### Steps To Reproduce

1. Enter a vehicle
2. Check Vehicle: Drive on water checkbox
3. Drive into water
4. Notice that vehicle does not drive *on* water

### Expected Behavior

Should be able to drive on water.

### Logs

No errors logged.

Injection shows:
[22:50:12.4135296][INFO/main.cpp:156] Settings Loaded and logger initialized.
[22:50:12.4135487][INFO/main.cpp:158] Yim's Menu Initializing
[22:50:12.4135500][INFO/main.cpp:159] Git Info
	Branch:	master
	Hash:	a9eaaebb42eab516262ac513db1ef44dcdc0dbcf
	Date:	Sat May 25 07:22:05 2024
[22:50:12.4136564][INFO/main.cpp:171] Windows 10 Pro Version 22H2 (OS Build 19045.4412)
[22:50:12.4180591][DEBUG/thread_pool.cpp:22] Resizing thread pool from 0 to 4
[22:50:12.4182351][INFO/main.cpp:179] Thread pool initialized.
[22:50:12.4186016][INFO/pointers.hpp:94] GTA5 pointers cache is up to date, using it.
[22:50:12.4186089][INFO/pointers.cpp:1878] Loading pointers instance from cache
[22:50:12.4186100][INFO/pointers.cpp:1887] Pointers cache: Loading 224 fields from the cache
[22:50:12.4186165][DEBUG/module.cpp:55] Waiting for socialclub.dll...
[22:50:12.4188158][INFO/pointers.hpp:94] Social Club pointers cache is up to date, using it.
[22:50:12.4188200][INFO/pointers.cpp:1878] Loading pointers instance from cache
[22:50:12.4188212][INFO/pointers.cpp:1887] Pointers cache: Loading 5 fields from the cache
[22:50:12.4188509][INFO/main.cpp:182] Pointers initialized.
[22:50:12.4188577][INFO/main.cpp:189] Disabled anticheat gameskeleton.
[22:50:12.4190315][INFO/main.cpp:192] Byte Patch Manager initialized.
[22:50:12.4195622][INFO/main.cpp:195] Renderer initialized.

### Store

Steam

### Language

English

### YimMenu Language

English (US)

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

**Comment by rkwapisz** [comment]
It sorta works for me. It's shitty, though.

**Comment by Rxann** [comment]
> It sorta works for me. It's shitty, though.

Yeah I noticed the shittyness too. I mean it works but my car got destroyed. idk if there is too much we can do about this.

**Comment by Flylow260** [comment]
you need to enable "drive under water" aswell, works just fine

**Comment by KernelFrog** [comment]
> you need to enable "drive under water" aswell, works just fine

Doesn't work for me.  :-(  What am I doing wrong?

**Comment by Flylow260** [comment]
> > you need to enable "drive under water" aswell, works just fine
> 
> Doesn't work for me. :-( What am I doing wrong?

So you have both activated and it doesn't work? Have you activated water in your car's immunities?
